apps/dashboard/services/graphql/queries/get-transactions.ts

```diff
@@ -0,0 +1,202 @@
+                  transactionHash
+                  vout
+                }
+              }
+              status
+              settlementPrice {
+                base
+                currencyUnit
+                formattedAmount
+                offset
+              }
+              initiationVia {
+                ... on InitiationViaIntraLedger {
+                  counterPartyUsername
+                  counterPartyWalletId
+                }
+                ... on InitiationViaOnChain {
+                  address
+                }
+                ... on InitiationViaLn {
+                  paymentHash
+                }
+              }
+            }
+          }
+          pageInfo {
+            endCursor
+            hasNextPage
+            hasPreviousPage
+            startCursor
+          }
+        }
+      }
+    }
+  }
+`;
+
+gql`
+  query GetFirstTransactions($first: Int) {
+    me {
+      id
+      defaultAccount {
+        transactions(first: $first) {
+          edges {
+            cursor
+            node {
+              createdAt
+              direction
+              id
+              memo
+              settlementAmount
+              settlementCurrency
+              settlementDisplayAmount
+              settlementDisplayCurrency
+              settlementDisplayFee
+              settlementFee
+              settlementVia {
+                ... on SettlementViaIntraLedger {
+                  counterPartyUsername
+                  counterPartyWalletId
+                }
+                ... on SettlementViaLn {
+                  paymentSecret
+                  preImage
+                }
+                ... on SettlementViaOnChain {
+                  transactionHash
+                  vout
+                }
+              }
+              status
+              settlementPrice {
+                base
+                currencyUnit
+                formattedAmount
+                offset
+              }
+              initiationVia {
+                ... on InitiationViaIntraLedger {
+                  counterPartyUsername
+                  counterPartyWalletId
+                }
+                ... on InitiationViaOnChain {
+                  address
+                }
+                ... on InitiationViaLn {
+                  paymentHash
+                }
+              }
+            }
+          }
+          pageInfo {
+            endCursor
+            hasNextPage
+            hasPreviousPage
+            startCursor
+          }
+        }
+      }
+    }
+  }
+`;
+
+export async function fetchFirstTransactions(
+  token: string,
+  first: number = 10,
+) {
+  const client = apollo(token).getClient();
+
+  try {
+    const data = await client.query<GetFirstTransactionsQuery>({
+      query: GetFirstTransactionsDocument,
+      variables: { first },
+    });
+    return data.data.me?.defaultAccount.transactions;
+  } catch (e) {
+    console.error("error", e);
+    if (e instanceof Error) {
```

Comment:
let's not use 1 letter variable name. it make the code harder to read if it's used too much.

```suggestion
    if (err instanceof Error) {
```